Assets/Project/Modules/Enemies/Slime/Scripts/SlimeRefactor/SquashStretchAnimator.cs

```diff
@@ -66,12 +68,16 @@ public void PlayDeath()
 
         public void PlayMove()
         {
+            //move animation
             _playAnimation = true;
+            _animator.SetBool("Moving", true);
             SquashAndStretch();
         }
 
         public void StopMove()
         {
+            //idle animation
+            _animator.SetBool("Moving",false);
```

Comment:
Make a constant `string const MOVE_ANIMATOR_PARAMETER = "Moving"`.
Although there are better ways than a constant, like:
A ScriptableObject class "SlimeEnemyAnimationConfig", with a getter property returning the Parameter to be used in the animator.